### PR TITLE
change label for Exit, Force Exit and Withdraw

### PR DESCRIPTION
**Pull request by mmilenkovic**
Could anyone advise if this change is correct? In terms of applying the same label for `Withdraw`, `Exit` and `ForceExit`?

### CODE REVIEW

File: src/views/transaction/transaction.view.jsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ function Transaction ({
       case TxType.Withdraw:
       case TxType.Exit:
       case TxType.ForceExit: {
-        return 'Withdraw'
+        return 'Exit (Initiate withdraw - step 1)'
```
Comment (AlbertoElias) [inline src/views/transaction/transaction.view.jsx:44]: I have concerns with the label itself. Maybe `Exit (Withdraw Step 1)` The Initiate makes it feel as if the tx hasn't been completed

Comment (mmilenkovic) [reply src/views/transaction/transaction.view.jsx:44]: We'll add "Withdraw Step 1/2 Completed" instead.

File: src/views/transaction/transaction.view.jsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ function Transaction ({
       case TxType.Withdraw:
       case TxType.Exit:
       case TxType.ForceExit: {
-        return 'Withdraw'
+        return 'Withdraw Step 1/2 Completed'
```
Comment (AlbertoElias) [inline src/views/transaction/transaction.view.jsx:44]: @chamian do you agree?

Comment (mmilenkovic) [reply src/views/transaction/transaction.view.jsx]: Latest changed added as per Chami's request.

Comment (mmilenkovic) [reply src/views/transaction/transaction.view.jsx]: Screenshot 2021-05-25 at 12 02 35

**Review by AlbertoElias — APPROVED** [review]

**Pull request merged**